easybuild/easyconfigs/s/SimPEG/SimPEG-0.14.1-intel-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,77 @@
+# Author: Pavel Grochal (INUITS)
+# License: GPLv2
+
+easyblock = 'PythonBundle'
+
+name = 'SimPEG'
+version = '0.14.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://simpeg.xyz/'
+description = """Simulation and Parameter Estimation in Geophysics 
+- A python package for simulation and gradient based parameter estimation in the context of geophysical applications."""
+
+toolchain = {'name': 'intel', 'version': '2020a'}
+
+dependencies = [
+    ('Python', '3.8.2'),
+    ('SciPy-bundle', '2020.03', versionsuffix),
+    ('matplotlib', '3.2.1', versionsuffix),
+    ('LLVM', '9.0.1'),
+]
+
+use_pip = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+exts_list = [
+    ('pyMKL', '0.0.3', {
+        'modulename': '%(name)s',
+        'checksums': ['5a24f63f510007d908506c88aa5b6b720a0d9201b8222d14805aca1f771b6afa'],
+    }),
+    ('pymatsolver', '0.1.2', {
+        'checksums': ['3ea58d5de6b0a05f646359766932f186d74d46b23730613f9fbff1482b2dde51'],
+    }),
+    ('properties', '0.6.1', {
+        'checksums': ['b119ce4c53f4717fa29371c1bb929f422dda7ed90a4559b3583ea77389177263'],
+    }),
+    ('vectormath', '0.2.2', {
+        'checksums': ['f9f9209d350ec89dc11e2548e0e6397c1b9489c8468eb50ce33788ee4322a7b8'],
+    }),
+    ('discretize', '0.4.12', {
+        'checksums': ['b90ba8ab32c4009c4b6af1962f27a15bc69d47b4d752eaf71c033be0551d2387'],
+    }),
+    ('utm', '0.5.0', {
+        'checksums': ['07e55707ed660eec1ae983bd54a406c437962618a6261b38d70592fe30f5f508'],
+    }),
+    ('geoana', '0.0.6', {
+        'checksums': ['7f37a72a722f5f0690fdefe0cbafc59d613143962e154c87210a0b4f6d1d3730'],
+    }),
+    ('llvmlite', '0.33.0', {
+        'checksums': ['9c8aae96f7fba10d9ac864b443d1e8c7ee4765c31569a2b201b3d0b67d8fc596'],
+    }),
+    ('numba', '0.50.0', {
```

Comment:
@Darkless012 would you mind creating a separate easyconfig for numba, so it can be reused by others as well?

Reply:
sure thing :)

Reply:
I made that separate PR, so it gets merged easier.
https://github.com/easybuilders/easybuild-easyconfigs/pull/10862